lib/commands/europeana/portal/docker/templates/docker-compose.services.yml

```diff
@@ -0,0 +1,19 @@
+version: "3"
+services:
+  minio:
+    image: minio/minio:RELEASE.2018-09-01T00-38-25Z
+    ports:
+    - "3001:9000"
```

Comment:
the ports here are the same as used by the europeana contribute docker services. While not necessarily a problem, since it's easy to start/stop the relevant docker instances, it might be nice to run everything here 10 or 100 ports up so that both applications can easily be run side by side.

Reply:
Good idea.